excercise-e-shop/js/cart.js

```diff
@@ -1,197 +1,55 @@
 //get total price
-var price = cart.reduce(function (price, product) {
+var price = getCart().reduce(function (price, product) {
   return price + product.price * product.quantity;
 }, 0)
```

Comment:
- vẫn còn tình trạng dùng biến global như thế này hả em? sửa lại giống như biến `cart` em nhé.
- Khi update thì phải lấy từ 1 source tin tưởng duy nhất đó là localStorage. Tìm hiểu concept `Single source of truth`